tests/SprykerTest/Zed/RabbitMq/Business/Model/Exchange/Filter/ExchangeFilterByNameTest.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * MIT License
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
```

Comment:
Please use updated code sniffer, that will then fix this to MIT automatically.
Also please update the MIT license header as per new specs.